.github/ISSUE_TEMPLATE/question.md

```diff
@@ -3,5 +3,5 @@ name  : Ask a Question
 about : The issue tracker is not for questions -- please ask questions at https://community.rstudio.com/c/shiny.
 ---
 
-The issue tracker is not for questions. If you have a question, please feel free to ask it on our community site, at https://community.rstudio.com/c/shiny.
+The issue tracker is not for questions. If you have a question, please feel free to ask it on our community site, at https://forum.posit.co/c/shiny.
```

Comment:
The link with `/c/` doesn't seem to exist?

```suggestion
The issue tracker is not for questions. If you have a question, please feel free to ask it on our community site, at https://forum.posit.co/tags/shiny.
```

Reply:
Does for me...the number reference at the end isn't required

<img width="324" alt="Screenshot 2024-05-01 at 11 36 55 AM" src="https://github.com/rstudio/shiny/assets/2762787/9558444b-771e-4dbe-9c25-8d96b5283feb">

Reply:
Ahh, whoops. I think I included the `.` at the end when testing it